.project-cid

```diff
@@ -1 +1 @@
-Qmdci3j6pCMW7YnPKE5MVRMe9EZ1ximDdepeSjKW4VgciV
\ No newline at end of file
+QmcYQmhy3gneCFqTaHfL44JdDYqxh8QVpEdXnu37ChrtFA
```

Comment:
@Douglasacost where do you get this from? How we can make sure it corresponds with this commit of the project?

Reply:
This is the last deployed version to IPFS, so it is a CID. Will add it to gitignore

Reply:
Good idea, let's remove this file because we cannot ensure it would contain the right CID.